strings/tips.txt

```diff
@@ -19,8 +19,8 @@ Some roles cannot be antagonists by default, but antag selection is decided firs
 There are many places around the station to hide contraband. A few for starters: linen boxes, toilet cisterns, body bags. Experiment to find more!
 You can use a machine in the vault to deposit cash or rob Cargo's department funds.
 When in doubt about technicial issues, clear your cache (byond launcher > cogwheel > preferences > game prefs), update your BYOND, and relog.
-Most things have special interactions with your middle mouse button, alt, shift, and control click. Experiment!
-You can screwdriver any non-chemical grenade to shorten fuses from 5 seconds, to 3 seconds, to 1 second, to instant boom! Nobody ever expects this.
+Most things have special interactions with alt, shift, and control click. Experiment!
+You can screwdriver any non-chemical grenade to shorten fuses from 5 seconds, to 3 seconds, to instant boom! Nobody ever expects this.
```

Comment:
are you sure this is still a thing? last time i played i remember it only being 3 and 5

Reply:
Yes, for non-chem nades.

Chem nades only have 3 and 5 now